Makefile

```diff
@@ -70,11 +70,15 @@ get-pgmonitor:
 	git -C '$(dir $(PGMONITOR_DIR))' clone https://github.com/CrunchyData/pgmonitor.git || git -C '$(PGMONITOR_DIR)' fetch origin
 	@git -C '$(PGMONITOR_DIR)' checkout '$(PGMONITOR_VERSION)'
 	@git -C '$(PGMONITOR_DIR)' config pull.ff only
+	[ -d /opt/crunchy/conf ] || sudo mkdir -p /opt/crunchy/conf
```

Comment:
I don't think we should reach outside the `postgres-operator` repo - can we update this to point somewhere in the repo, maybe `hack/` something?